packages/atomic/src/pages/accessibility-b.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic</title>
+
+    <script type="module" src="/build/atomic.esm.js"></script>
+    <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/themes/accessible.css" />
+    <script>
+      (async () => {
+        await customElements.whenDefined('atomic-search-interface');
+        const searchInterface = document.querySelector('#search');
+
+        await searchInterface.initialize({
+          accessToken: 'xxc23ce82a-3733-496e-b37e-9736168c4fd9',
```

Comment:
this access token is already public, I was able to see it from the demo on both my work computer and my personal computer without logging into anything.